test/close.test.js

```diff
@@ -1,11 +1,14 @@
 'use strict'
 
+const { Server } = require('http')
 const net = require('net')
 const t = require('tap')
 const test = t.test
 const Fastify = require('..')
 const { Client } = require('undici')
 
+const nodeSupportsConnectionsClosing = 'closeAllConnections' in Server.prototype
```

Comment:
Quick one, it shouldn't be better to rely on the Node version, or there're plans to backport it to previous versions?

Reply:
detecting support this way is actually simpler.

Reply:
Moreover as I'm not sure about plans on Node v14 and v16, I chose to detect actual support without hardcoding versions.